tests/sentry/sentry_metrics/test_batch.py

```diff
@@ -227,36 +231,184 @@ def test_extract_strings_with_rollout(should_index_tag_values, expected):
     assert batch.extract_strings() == expected
 
 
-def test_all_resolved(caplog, settings):
-    settings.SENTRY_METRICS_INDEXER_DEBUG_LOG_SAMPLE_RATE = 1.0
+def test_extract_strings_with_multiple_use_case_ids():
+    perf_distribution_payload = {
+        "name": TransactionMRI.MEASUREMENTS_FCP.value,
+        "tags": {
+            "environment": "production",
+            "session.status": "healthy",
+        },
+        "timestamp": ts,
+        "type": "d",
+        "value": [4, 5, 6],
+        "org_id": 1,
+        "retention_days": 90,
+        "project_id": 3,
+    }
     outer_message = _construct_outer_message(
         [
             (counter_payload, []),
-            (distribution_payload, []),
+            (perf_distribution_payload, []),
             (set_payload, []),
         ]
     )
-
     batch = IndexerBatch(
-        UseCaseKey.PERFORMANCE,
         outer_message,
         True,
         False,
         arroyo_input_codec=_INGEST_SCHEMA,
     )
-    assert batch.extract_strings() == (
-        {
+    assert batch.extract_strings() == {
+        "release-health": {
             1: {
                 "c:sessions/session@none",
-                "d:sessions/duration@second",
                 "environment",
                 "errored",
-                "healthy",
                 "init",
                 "production",
                 "s:sessions/error@none",
                 "session.status",
             }
+        },
+        "performance": {
```

Comment:
Even though this is for testing only, a reader could believe that release health and performance traffic can flow through a single pipeline. Which is not possible. Is there a way not to bake that assumption in our tests as well?

Reply:
Fixed now